docs/public/static/error-codes.json

```
@@ -17,5 +17,9 @@
   "16": "MUI: withTheme is no longer exported from @mui/material/styles.\nYou have to import it from @mui/styles.\nSee https://mui.com/r/migration-v4/#mui-material-styles for more details.",
   "17": "MUI: Expected valid input target. Did you use a custom `slots.input` and forget to forward refs? See https://mui.com/r/input-component-ref-interface for more info.",
   "18": "MUI: `vars` is a private field used for CSS variables support.\nPlease use another name.",
-  "19": "MUI: `useColorScheme` must be called under <CssVarsProvider />"
-}
+  "19": "MUI: `useColorScheme` must be called under <CssVarsProvider />",
+  "20": "MUI: The `experimental_sx` has been moved to `theme.unstable_sx`.For more details, see https://github.com/mui/material-ui/pull/35150",
+  "21": "MUI: The `experimental_sx` has been moved to `theme.unstable_sx`.To migrate, see https://mui.com/material-ui/customization/theme-components/#the-sx-syntax-experimental",
+  "22": "MUI: The `experimental_sx` has been moved to `theme.unstable_sx`.You should remove the import and replace the usage with `theme.unstable_sx(…same value you pass to experimental_sx)` instead.",
+  "23": "MUI: The `experimental_sx` has been moved to `theme.unstable_sx`.For more details, see https://github.com/mui/material-ui/pull/35150."
```

```suggestion
  "20": "MUI: The `experimental_sx` has been moved to `theme.unstable_sx`.For more details, see https://github.com/mui/material-ui/pull/35150."
```

I believe this is sufficient now.

---

Thanks, my bad. I thought that the script would reset and collect the new one.